lib/mocha.js

```diff
@@ -402,6 +402,18 @@ Mocha.prototype.noHighlighting = function() {
 };
 
 /**
+ * Enable uncaught errors to propagate (in browser).
+ *
+ * @return {Mocha}
+ * @api public
+ */
+
+Mocha.prototype.allowUncaught = function(){
+  this.options.allowUncaught = global.mocha.allowUncaught = true;
```

Comment:
Yes, in fact `global.mocha.allowUncaught` doesn’t need to be set at all. The global error handler will still work fine this way in the browser.